ote/src/cljs/ote/app/routes.cljs

```diff
@@ -24,9 +26,14 @@
     ["/route/new" :new-route]
     ["/edit-route/:id" :edit-route]
 
+    ["/routes/view-gtfs" :view-gtfs]
```

Comment:
I think that you could add this route to admin panel also (maybe not in this pr but soon) ?